pymc/backends/arviz.py

```diff
@@ -240,7 +241,9 @@ def __init__(
 
         self.observations = find_observations(self.model)
 
-    def split_trace(self) -> Tuple[Union[None, "MultiTrace"], Union[None, "MultiTrace"]]:
+    def split_trace(
```

Comment:
Was the line length changed, which causes this line to now split? Can that be set to the old value?

Reply:
I did keep `line-length = 100` and it is not the `ruff` but the `ruff formater`. I am following the recommended configuration. i will investgate.

Reply:
Then may be fine. Changes otherwise look good!

Reply:
Let's see if https://github.com/pymc-devs/pymc/pull/7091/commits/bf843a4dfef3df67fce106b59bdf4bc3cf26850b works 🤞

Reply:
It worked! Do you want me to revert any other line?

Reply:
> It worked! Do you want me to revert any other line?

Yeah that would be great